docs/Makefile

```diff
@@ -18,6 +18,9 @@ publish: clean
 	$(MAKE) build
 	$(MAKE) -C deploy publish
 
+live:
+	npx nodemon --ext txt --exec 'make build'
```

Comment:
how do we know for sure that everyone has this tool? It doesn't seem like we have this anywhere else

Reply:
technically this would rebuild for changes to the requirements file.. but not sure if that's fine with us

also, does this go down into directories too?

Reply:
it does go into directories by default. npx comes bundled with NPM & Node which is already a dependency of our build system (is also directly used in WebUI build)

> technically this would rebuild for changes to the requirements file.. but not sure if that's fine with us

good point, we could probably add some rules to avoid such cases but considering: we are able to build anytime, doc builds are fast, we don't expect requirements.txt to change often, etc I think we'd be find by keeping it simple. 

they give us this option:   `-i, --ignore ............. ignore specific files or directories.`

Reply:
also open to different target names, just keeping it the same as the other examples I could find for this (webui react and elm). 

we could potentially add this for master changes to `go` and tac on reloading the master as well :thinking:

Reply:
re: doing it for the master, I think we should hold off since we would need to restart the cluster, so the addition of the automatic rebuild doesn't really buy us much

Reply:
it looks like this is running `make build` whenever any `txt` file within the `docs/` directory changes.

this might not be frequent but making it detect changes to the determined sphinx theme:
basically adding `html` or `js` to the `--ext` list.